### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#1793] [Ideas]
Hello, where can we manage translation?

I think about the 'required' in the forms and the date field.

Thank you

**Comment by mjashanks** [comment]
Yes - I see what you mean. The validation message is always in English, when submitting a form.

Unfortunately not possible at the moment - but we really should allow for this. 

Linking to this: https://github.com/Budibase/budibase/discussions/1679 as it is related.

Thanks!
